.murdock

```diff
@@ -125,7 +125,7 @@ get_app_board_toolchain_pairs() {
 
     # collect extra arguments into prefix variable
     shift
-    local prefix="$@"
+    local prefix="$*"
```

Comment:
I believe the `$@` => `$*` change accidentally was added to 1818da495262fb9ed6d39dc5b775bd25d116d6f0.

Reply:
the change is necessary. it belongs to the first commit, though... I'll fix that.

Reply:
done